core/benchmarks/MapStatusesConvertBenchmark-results.txt

```diff
@@ -0,0 +1,13 @@
+================================================================================================
+MapStatuses Convert Benchmark
+================================================================================================
+
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_281-b09 on Mac OS X 10.15.7
```

Comment:
Could you update the benchmark result using Github action? https://github.com/apache/spark/actions/workflows/benchmark.yml

Reply:
Hi, @wangyum I just updated the benchmark result file. 

Thanks